### DISCUSSION TITLE
[@types/aws-lambda] Missing property enableSMSMFA in BaseUserMigrationTriggerEvent

### DISCUSSION THREAD

**Discussion by andrebask** [DefinitelyTyped/DefinitelyTyped#64182] [Issues with a @types package]
Hi,
I think the `BaseUserMigrationTriggerEvent` interface [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dd71e27208128c3011c6d17788ec18ef7de07e06/types/aws-lambda/trigger/cognito-user-pool-trigger/user-migration.d.ts#LL13C18-L13C47) is missing the `enableSMSMFA` property documented [here](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-migrate-user.html#cognito-user-pools-lambda-trigger-syntax-user-migration)?

Thanks

[a comment by an automated account is not shown]

**Comment by jamesorlakin** [comment]
Hi @andrebask,

Hmm, yeah that's probably a recent change. Looking back at the AWS docs it was brought in a few months ago: https://github.com/awsdocs/amazon-cognito-developer-guide/commit/becc0bb9fd617446d5bda4a3936f12d783e5b2c7#diff-e7dad554e45de54fd5ad0b7b014802b8b8a8ec355884890e12c8e298c5dfe219R50

If you're able to make a PR it'd be much appreciated! If not, no worries - I can maybe take a look at this when I'm back from holidays in a week. I may need a reminder 😉 

Thanks.

**Comment by ijborda** [comment]
Fixed here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67526
